src/style.css

```diff
@@ -0,0 +1,120 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  --card__link-color: #00acdc;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 30px 14px 15px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 45px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+}
+
+.card__name-product {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin-bottom: 6px;
+}
+
+.card__code-producte {
+  font-size: 11px;
+  line-height: 13px;
+  margin-bottom: 16px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+.card__stars {
+  display: flex;
+  align-items: center;
+}
+
+.card__star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background-image: url(../src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.card__star_active {
+  background-image: url(../src/images/star-active.svg);
+}
+
+.card__count {
+  align-self: center;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card__cost {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.card__price-text {
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card__price {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.card__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-decoration: none;
+  color: #fff;
+  background-color: var(--card__link-color);
+  border-radius: 5px;
+  padding-top: 12px;
+  padding-bottom: 12px;
+}
+
+.card__link:hover {
+  background-color: #fff;
+  color: var(--card__link-color);
+  border: 1px solid var(--card__link-color);
```

Comment:
at this moment when you hovered on your link - the border appears and adds 1px, because of box-sizing: border-box; width and height of your button is recalculating, and text could change its position, so it's better if you write border property inside the button.